usermanagerlib/src/main/java/org/pesmypetcare/usermanagerlib/datacontainers/Medication.java

```diff
@@ -0,0 +1,84 @@
+package org.pesmypetcare.usermanagerlib.datacontainers;
+
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+
+public class Medication {
+    private String date;
+    private String name;
+    private MedicationData body;
+
+    public Medication(String date, String name, MedicationData body) {
+        this.date = date;
+        this.name = name;
+        this.body = body;
+    }
+
+    /**
+     * Returns the date of the Medication
+     * @return Medication date
+     */
+    public String getDate() {
+        return this.date;
+    }
+
+    /**
+     * Returns the name of the Medication
+     * @return Meadication name
+     */
+    public String getName() {
+        return this.name;
+    }
+
+    /**
+     * Sets a new date
+     * @param date medication date
+     */
+    public void setDate (String date) {
+        this.date = date;
+    }
+
+    /**
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [First sentence should end with a period.](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care-Libraries/pullRequest?prid=5403182)